.github/workflows/check-commit.yml

```diff
@@ -6,6 +6,8 @@ on:
 
 jobs:
   check-signed-off-by:
+    if: ${{ ! contains(github.event.pull_request.labels.*.name, 'PR/NO TEST') }}
```

Comment:
For reviews,
- in yml, single quote(') doesn't parse escape characters like `\n` and contains space(' ').

Reply:
(I was not sure so I googled.)